Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task1/sweets/Gift.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.cdp.volha_ivanova.task1.sweets;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.chocolateCandies.ChocolateCandies;
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.lollipop.Lollipop;
+import com.epam.cdp.volha_ivanova.task1.sweets.chocolate.Chocolate;
+import com.epam.cdp.volha_ivanova.task1.sweets.waffles.Waffles;
+
+public class Gift {	
+	
+	int giftWeight;			
+	Scanner scanner = new Scanner(System.in);
+	Waffles waffle = new Waffles("Waffles", 10, scanner.nextInt());	
```

Comment:
I suppose creation of default gift should be in separate class. for example GiftFactory and method createDefaultGift()